com.unity.perception/Runtime/GroundTruth/Labelers/CameraLabeler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    [Serializable]
+    public abstract class CameraLabeler
```

Comment:
Some of the "labelers" that inherit from this class don't seem to be doing any labeling. ObjectCountLabeler and RenderedObjectInfoLabeler are consuming labels to produce metadata, but do no labeling of their own, right? It seems like, similar to how the Labelers are added as "sub-components" (I don't know what the right word is) of PerceptionCamera, these MetadataGenerators(™️) could be added as sub-components to the passes whose labels they're consuming to create their data.

Reply:
As we discussed, we'll keep it unless we can come up with a great name :)